pyproject.toml

```diff
@@ -33,24 +33,43 @@ version = "3.0.0.dev0"
 
 [project.optional-dependencies]
 docs = [
-        "numpy",
-        "numpydoc",
-        "matplotlib",
-        "pillow",
-        "sphinx",
-        "sphinx_rtd_theme",
-        "sphinx-gallery",
-        "sphinxcontrib-napoleon",
-        "sphinx-copybutton"
+    "numpy",
+    "numpydoc",
+    "matplotlib",
+    "pillow",
+    "sphinx",
+    "sphinx_rtd_theme",
+    "sphinx-gallery",
+    "sphinxcontrib-napoleon",
+    "sphinx-copybutton"
 ]
+
 dev = [
     "black==23.3.0",
     "mypy==1.4.1",
+    "pydicom-data",
     "pytest",
     "pytest-cov",
     "ruff==0.0.277",
+    "types-requests"
 ]
 
+basic = ["numpy"]
+
+pixeldata = [
+    "numpy",
+    "pillow", # leave out? Better alternatives below
+    "pyjpegls",
+    "pylibjpeg[openjpeg]",
+    "pylibjpeg[rle]",
+    "python-gdcm"
+]        
+
+# Consider libjpeg-turbo (BSD-like license) when 12-bit available
+# https://github.com/libjpeg-turbo/libjpeg-turbo/issues/199
+gpl-license = ["pylibjpeg[libjpeg]"]
```

Comment:
Good point. If not for the GPL with `libjpeg`, we could have covered all bases with the `pylibjpeg` plugins.